sql/expression/function/json_extract.go

```diff
@@ -69,10 +69,12 @@ func (j *JSONExtract) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 			return nil, err
 		}
 
-		result[i], err = jsonpath.JsonPathLookup(doc, path.(string))
+		c, err := jsonpath.Compile(path.(string))
 		if err != nil {
 			return nil, err
 		}
+
+		result[i], _ = c.Lookup(doc) // err ignored
```

Comment:
Maybe it's worth it to give some info to the user in this case using `Warn`:

```go
ctx.Warn(0, err.Error())
```

Reply:
I believe that will be noise, because can be very common, since for example in my `LOC` if the language is not supported will be empty and then a warm